lib/player.rb

```diff
@@ -1,8 +1,7 @@
 # frozen_string_literal: true
 
 class Player
-  attr_reader :name, :marker
-  attr_writer :name
+  attr_accessor :name, :marker
```

Comment:
It was good as it was before, it's recommended not to use `attr_accessor` for this project.
-------------
The thing was that it would be better not to use `attr_writer :name` since it is not necessary to use a setter as you are doing in :

https://github.com/addod19/OOP_TicTacToe/blob/36316d67b122e8695a0b125b352d1bccda080a0c/bin/main.rb#L18
https://github.com/addod19/OOP_TicTacToe/blob/36316d67b122e8695a0b125b352d1bccda080a0c/bin/main.rb#L19

since you could use something like `player1 = Player.new(player_names[0])` or in your case `player1 = Game.new(player1, player2) ` giving some arguments to the initializer.

So you can choose this way or the way you were using before, correct this before merging :+1: